manual/Creating_a_table/NLP_and_tokenization/CJK.md

```diff
@@ -1,8 +1,8 @@
-# Chinese, Japanese and Korean (CJK) languages
+# Chinese, Japanese and Korean (CJK) and Thai languages
 
-Manticore provides built-in support for indexing CJK texts, allowing you to process CJK texts in two different ways:
+Manticore provides built-in support for indexing languages with [continuous scripts](https://en.wikipedia.org/wiki/Scriptio_continua) (i.e., languages that does not use spaces or other marks between the words or sentences), allowing you to process texts in these languages in two different ways:
```

Comment:
Manticore provides built-in support for indexing languages with [continuous scripts](https://en.wikipedia.org/wiki/Scriptio_continua) (i.e., languages that do not use spaces or other marks between words or sentences), allowing you to process texts in these languages in two different ways: